versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java

```diff
@@ -524,4 +524,39 @@ private static void checkKeysAndValuesIndividually(
           .isEqualTo(valueGen.apply(i).serialized());
     }
   }
+
+  public static Stream<List<String>> progressiveKeyNames() {
+    return IntStream.range(1, 100)
+        .mapToObj(i -> IntStream.range(1, i).mapToObj(j -> "a-" + j).collect(Collectors.toList()));
```

Comment:
You probably want `rangeClosed` here

Reply:
1 to 99 is enough, 100 is not a hard limit more of a "large enough" number :shrug: